MiniAn/minian_main.py

```diff
@@ -431,7 +432,112 @@ def load_doric_to_xarray(
 
     return varr, file_
 
-
+            
+def cross_register(minian_parameters, currentFile_AC, currentFile_A):
```

Comment:
I put `minian_parameters` I don't know if it is a smart move... but can do that for this function too? So it is uniform with the other functions